### PR TITLE
[Fixes]"payment_preference" field to Member model

### PR DESCRIPTION
**Pull request by harsha-mangena**
- refer https://github.com/nycmeshnet/meshdb/issues/394
- added attribute `payment_preference` on member
- added testcase

### CODE REVIEW

File: src/meshapi/models/member.py [hunk 1]
```diff
@@ -41,6 +41,7 @@ class Member(models.Model):
         "import process and original spreadsheet data. However this structure can be changed by admins at any "
         "time and should not be relied on by automated systems. ",
     )
+    payment_preference = models.CharField(choices=('cash', 'stripe', None), default=None)
```
Comment (WillNilges) [inline src/meshapi/models/member.py:44]: Do you mind creating a `models.TextChoices` class for this (similar to what's used in [node.py](https://github.com/nycmeshnet/meshdb/blob/main/src/meshapi/models/node.py#L17-L20)?)

**Review by WillNilges — CHANGES_REQUESTED** [review]
Thanks for the contribution! I let the CI checks run, and it looks like you need to run our linters and makemigrations. You can find instructions [here](https://github.com/nycmeshnet/meshdb?tab=readme-ov-file#invokepy-commands).

I also left a suggestion for how to implement the payment preferences choices.

**Comment by WillNilges** [comment]
Looks like you've got a syntax error causing the CI to fail. Can you fix that?

**Comment by Andrew-Dickinson** [comment]
Closing this due to inactivity. Feel free to re-open if you want to continue working on this

**Pull request closed without merging**